src/UnifiedFFI-Tests/FFIValueHolderTest.class.st

```diff
@@ -151,6 +160,37 @@ FFIValueHolderTest >> testCallIntPointerWithValueHolder [
 	holder value free
 ]
 
+{ #category : 'tests' }
+FFIValueHolderTest >> testCallStructPointerPointerWithValueHolder [
+	| holder result string |
+	
+	holder :=  FFITestStructure newValueHolder.
+	
+	FFICallbackFunctionResolution 
+		registerCallback: (FFICallback 
+			signature: #(int (FFITestStructure **aValue)) 
+			block: [ :aValue |
+				| bytes allocated |
+				bytes := ByteArray streamContents: [ :stream |
+					0 to: 2 do: [ :i | 
+						stream nextPutAll: (FFITestStructure new 
+							long: 42 + i;
+							getHandle) ] ].
+				allocated := ExternalAddress allocate:bytes size.
+				LibC memCopy: bytes to: allocated size: bytes size.
+				aValue pointerAt: 1 put: allocated.
+				3 ])
+		as: #TestStructPointerPointerValueHolder.
+
+	result := self ffiMethodStructPointerPointer: holder.
+	
+	self assert: result equals: 3.
+	self 
+		assert: ((holder arrayOfSize: 3) collect: #long) 
+		equals: #(42 43 44).
```

Comment:
Here is something else that is tricky.
You have a `FFITestStructure newValueHolder`, which in my view is a value holder for `FFITestStructure`.
But here you are using it as a value holder for *an array of those!*.

What I understand is that the type of the holder is  `FFITestStructure *`, right?

This just makes the test veery difficult to understand.